packages/testing/src/execution_testing/config/env.py

```
@@ -20,22 +20,38 @@
 - Access configuration values via properties (e.g., EnvConfig().remote_nodes).
 """
 
+import os
 from pathlib import Path
 from typing import Dict, List
 
 import yaml
 from pydantic import BaseModel, HttpUrl, ValidationError
 
-
 def get_project_root() -> Path:
-    """Find project root by locating .git directory."""
+    """
+    Find project root by locating .git directory, or use alternatives when missing.
+
+    Priority:
+    1. Use EEST_PROJECT_ROOT environment variable if set.
+    2. Traverse upward to find .git directory (development use).
+    3. Fallback to current working directory if no .git found (e.g., in packages or CI).
+    """
+    
+    # 1. Environment Variable Check
+    env_root = os.environ.get("EEST_PROJECT_ROOT")
+    if env_root:
+        return Path(env_root).resolve()
+
+    # 2. .git Directory Check
     current = Path(__file__).resolve().parent
     while current != current.parent:
         if (current / ".git").exists():
             return current
         current = current.parent
-    raise FileNotFoundError("Could not find project root (no .git directory)")
-
+    
+    # 3. Fallback to CWD
+    print("Warning: .git directory not found. Falling back to current working directory.")
+    return Path.cwd().resolve()
```

Personally, I'd raise an exception rather than fallback to the current directory. It's too easy to create random `env.yaml`.

---

Hey @SamWilsn, thanks for the review!

That's a valid concern about the security risk. I went with the CWD fallback because of the user's intent: they are explicitly running eest make env.

For anyone using a packaged/non-git install, if we raise an exception, that command is effectively broken unless they know to manually set EEST_PROJECT_ROOT. The CWD fallback makes make env usable out-of-the-box and ensures the file is created where they ran the command, fulfilling their request.

If the team (cc @danceratopz) prefers strict compliance over out-of-the-box usability for the make env command, I'm happy to switch to raising an exception instead. Just let me know the policy direction!